pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -876,3 +876,13 @@ func (c updateRulesOrderInGroup) Exec(sess *xorm.Session, migrator *migrator.Mig
 	}
 	return nil
 }
+
+func AddForErrorColumnMigration(mg *migrator.Migrator) {
+	if !mg.Cfg.UnifiedAlerting.IsEnabled() {
```

Comment:
Do we need this? I can't see it in `AddAlertRuleMigrations` and `AddAlertRuleVersionMigrations`?